src/Service.Host/client/src/components/PurchaseOrders/CreatePurchaseOrderUt.js

```diff
@@ -526,12 +527,14 @@ function CreatePurchaseOrderUt() {
                                 label="Our price (currency)"
                                 propertyName="ourUnitPriceCurrency"
                                 onChange={(propertyName, newValue) =>
-                                    handleDetailValueFieldChange(
-                                        propertyName,
-                                        'baseUnitPrice',
-                                        newValue,
-                                        detail
-                                    )
+                                    dispatch({
+                                        payload: {
+                                            lineNumber: 1,
+                                            fieldName: 'propertyName',
```

Comment:
this will break this InputField since there is no property with fieldName 'propertyName' on a purchaseOrderDetail.